jme3-core/src/main/java/com/jme3/audio/AudioNode.java

```diff
@@ -113,6 +114,26 @@
     }
 
     /**
+     * <code>Type</code> indicates how to retrieve the audio data.
+     * It replaced the old "stream" and "streamCache" parameters.
+     * It defines whether the whole file is read and buffered or
+     * if it is read gradually from disk.
+     */
+    public enum Type {
```

Comment:
The type is a property of `AudioData`, not `AudioNode`. 
See for example, `AudioData.DataType` enum which has the same purpose. Perhaps it should just be better documented.